src/Table2Beta/Table.tsx

```diff
@@ -509,7 +525,12 @@ export class Table2Beta extends React.Component<Props, State> {
           <Menu
             trigger={
               <Button size="small" className={cssClass.ACTION_MENU} type="link">
-                <img className={cssClass.SINGLE_ACTION_TRIGGER} src={require("./ellipsis.svg")} />
+                <img
+                  className={cssClass.SINGLE_ACTION_TRIGGER}
+                  src={require("./ellipsis.svg")}
+                  // TODO: alt-texts need to be translated
+                  alt="more actions"
```

Comment:
is this text good?

Reply:
I'm not exactly sure the context, but maybe "More options"? I don't know the range of "actions" being used here, but the word "actions" feels like it's tied to the implementation and maybe not as clear to the user (who doesn't know that the button is an `ACTION_MENU`).